client/state/account-recovery/settings/test/actions.js

```diff
@@ -357,40 +330,6 @@ describe( 'account-recovery actions', () => {
 				type: ACCOUNT_RECOVERY_SETTINGS_RESEND_VALIDATION,
 				target: 'email',
 			} ),
-		postConditionSuccess: () =>
-			expect( spy ).toHaveBeenCalledWith( {
-				type: ACCOUNT_RECOVERY_SETTINGS_RESEND_VALIDATION_SUCCESS,
-				target: 'email',
-			} ),
-		postConditionFailed: () =>
-			expect( spy ).toHaveBeenCalledWith(
-				expect.objectContaining( {
-					type: ACCOUNT_RECOVERY_SETTINGS_RESEND_VALIDATION_FAILED,
-					target: 'email',
-					error: expect.objectContaining( errorResponse ),
-				} )
-			),
-	} );
```

Comment:
Ideally we should modify the `postCondition`s instead of deleting them. The tested thunks dispatch the `NOTICE_CREATE` actions on success/failure, with a certain `status` and `text` that can be tested.